spec/requests/all_casa_admins/casa_admins_spec.rb

```diff
@@ -21,3 +21,4 @@
     end
   end
 end
+# TODO: test admin creation as an all_casa_admin
```

Comment:
I see this TODO and I'm going to merge this anyway. Please make a new PR with new work :)

Reply:
Admins don't actually have any invite email sent to them. Should I add this feature and can it use the same template as volunteers/admins? I'm also not sure if this is a bad time to work on stuff since the Power team will start soon.

> A CASA/Prince George’s County CasaAdmin console account has been created for you. This console is for logging the time you spend and actions you take on your CASA case. You can log activity with your CASA youth, their family members, their foster family or placement, the DSS worker, your Case Supervisor and others associated with your CASA case (such as teachers and therapists).
 --
> Your console account is associated with this email. If this is not the correct email to use, please stop here and contact your Case Supervisor to change the email address. If you are ready to get started, please set your password. This is the first step to accessing your new CasaAdmin account.
>

Reply:
Adding an email for CasaAdmin invites is a feature I want! I don't know whether working right now is a good idea but if you get a PR open I will try to merge it 😅